app/models/manageiq/providers/amazon/cloud_manager/vm_or_template_shared/scanning.rb

```diff
@@ -26,10 +26,12 @@ def perform_metadata_scan(ost)
     @s3                    = ext_management_system.connect(connect_args)
     raise "Unable to obtain a new S3 resource" unless @s3
     ssaq_args              = {}
-    ssaq_args[:ssa_bucket] = ext_management_system.guid
-    ssaq_args[:region]     = ext_management_system.provider_region
-    ssaq_args[:sqs]        = @sqs
-    ssaq_args[:s3]         = @s3
+    ssaq_args[:ssa_bucket]    = "#{AmazonSsaSupport::DEFAULT_BUCKET_PREFIX}-#{ext_management_system.guid}"
+    ssaq_args[:request_queue] = "#{AmazonSsaSupport::DEFAULT_REQUEST_QUEUE}-#{ext_management_system.guid}"
+    ssaq_args[:reply_queue]   = "#{AmazonSsaSupport::DEFAULT_REPLY_QUEUE}-#{ext_management_system.guid}"
```

Comment:
This key generation is duplicated in the amazon_ssa_agent.  It would be preferable if there were a method on AmazonSsaSupport that returned the value(s), thus keeping the key generation in a single place, in case we have to change it later.